### PR TITLE
fix: restored unshared view + adaptation

### PR DESCRIPTION
**Pull request by GoOz**
Restored unshared error view that was stripped from @goldoraf refactoring and added some design  adaptation for mobile.

### CODE REVIEW

**Comment by enguerran** [comment]
[(3) [PHO] Ajouter des photos à un album existant depuis l'album](https://trello.com/c/UYFoR5R7/340-3-pho-ajouter-des-photos-%C3%A0-un-album-existant-depuis-lalbum)

File: src/public/App.jsx [hunk 1]
```diff
@@ -64,8 +65,22 @@ class App extends Component {
...
+      this.setState({error: 'Fetch error'})
+    }
+  }
+
   render () {
     const { t, album, photos } = this.props
+    if (this.state.error) {
+      return (
+        <div className={classNames(styles['pho-public-layout'], styles['pho-public-layout--full'])}>
+          <ErrorShare errorType={`public_album_unshared`} />
```
Comment (enguerran) [inline src/public/App.jsx]: A `fetchStatus === 'failed'` could have other causes but this one.

In `componentWillReceiveProps` you should add test like in this PR https://github.com/cozy/cozy-drive/pull/336/files#diff-77a0342c324df1795ff84f7011d10e78R32

File: src/public/index.styl [hunk 2]
```diff
@@ -8,6 +8,17 @@
     width 100%
     top-padded(30px)
 
+@media (max-width: (768/basefont)rem)
+    .pho-public-layout--full
```
Comment (enguerran) [inline src/public/index.styl]: How the #156 could have change CSS?

Comment (GoOz) [inline src/public/index.styl]: It didn't but since this PR is about the errorShare view and this fix an issue about errorShare i put it there.

**Pull request merged**